lib/rspec/rails/matchers/active_job.rb

```diff
@@ -174,7 +176,7 @@ def queue_adapter
         # @private
         class HaveEnqueuedJob < Base
           def initialize(job)
-            super()
+            super("enqueue", "enqueued")
```

Comment:
I could be argued either way, but I think instead of parameters, adding the 
```ruby
@verb_present_tense = verb_present_tense
@verb_past_tense = verb_past_tense
```
after the super here would be preferred.